packages/js-dash-sdk/src/SDK/Client/Platform/methods/documents/get.ts

```diff
@@ -14,8 +15,8 @@ declare interface fetchOpts {
     where?: WhereCondition[];
     orderBy?: OrderByCondition[];
     limit?: number;
-    startAt?: number;
-    startAfter?: number;
+    startAt?: number|Document|Identifier;
```

Comment:
Shall we accept string (base58) and Buffer? And convert everything to Buffer before we send it to DAPI?